### ISSUE TITLE
Symbol Tabel

### ISSUE THREAD

**Issue by sho94os** [mike-lischke/antlr4-c3#89]
Hi,
does the port c++ come with symbol tabel?

**Comment by mike-lischke** [comment]
No, it contains no port of the symbol table class, but you can take the one I wrote years ago for [MySQL Workbench](https://github.com/mysql/mysql-workbench/tree/8.0/library/parsers).

**Comment by sho94os** [comment]
@mike-lischke thanks for the answer. can you also show a small demo program where i can use the symbol tabel. because this has no documentation

**Comment by mike-lischke** [comment]
You can look at the [symbol table tests](https://github.com/mike-lischke/antlr4-c3/blob/master/tests/SymbolTable.spec.ts), which are written in TypeScript, but pretty well understandable.
